src/posix/platform/openthread-core-posix-config.h

```diff
@@ -81,4 +81,18 @@
 #ifndef OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_ENABLE
 #define OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_ENABLE 1
 #endif
+
+// Platform udp and platform netif is necessary for daemon mode to work.
```

Comment:
```suggestion
```
Daemon mode doesn't rely on these twos features. But I think it a good idea to enable them by default in this mode.